server/tests/unit/dataset/test_cxg_dataset.py

```diff
@@ -30,6 +30,9 @@ def test_schema_filters_int64(self):
         schema = data.get_schema()
         with open(f"{FIXTURES_ROOT}/schema_without_int64.json", "r") as json_file:
             expected_schema = json.load(json_file)
+
+            # remove uns from schema comparison until test dataset is updated
```

Comment:
I might be missing something here, @kaloster 😆 Doesn't this PR update the test spatial dataset?

Reply:
I think I meant we need to update the expected schema which comes from `schema_without_int64.json` which doesn't have `uns` - otherwise the test will fail